### PR TITLE
Fixing unstyled buttons

### PR DESCRIPTION
**Pull request by Emily-ejag**
## Types of changes

What types of changes does this pull request introduce?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (non-breaking change that does not add functionality but makes code cleaner or more efficient)
- [ ] Documentation Update
- [ ] Tests (new or updated tests)
- [ ] Style (changes to code styling)
- [ ] CI (continuous integration changes)
- [ ] Repository Maintenance
- [ ] Other (please describe below)

## Checklist

- [x] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
- [x] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
- [ ] Linting checks pass with my changes.
- [ ] Any existing unit tests pass with my changes.
- [ ] Any existing end-to-end tests pass with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
- [ ] I have added JSDoc comments as appropriate.
- [ ] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).
- [ ] I have shared this PR on the roar-pr-reviews channel (if I have access)
- [ ] I have linked relevant issues (if any)

### CODE REVIEW

File: src/pages/NotFound.vue [hunk 1]
```diff
@@ -3,7 +3,11 @@
     <h1>{{ $t('notFound.pageNotFound') }}</h1>
     <p>{{ $t('notFound.sorry') }}</p>
     <router-link :to="{ name: 'Home' }">
-      <PvButton :label="$t('notFound.label')" icon="pi pi-home" />
+      <PvButton
+        :label="$t('notFound.label')"
```
Comment (lucasxsong) [inline src/pages/NotFound.vue:7]: Could I suggest that the notFound.label be replaced with `Return to Home`? The original text is a bit wordy.

Comment (Emily-ejag) [reply src/pages/NotFound.vue:7]: Oh I can change that, I have to also change the Spanish label

Comment (lucasxsong) [reply src/pages/NotFound.vue:7]: Thank you!

**Review by lucasxsong — APPROVED** [review]
